### DISCUSSION TITLE
Beta Feedback

### DISCUSSION THREAD

**Discussion by bauerj** [bauerj/paperless_app#64] [Show and tell]
We regularly publish app updates as a beta version first. 

If you would like to help us in testing new features, please enroll in the public beta [here](https://play.google.com/apps/testing/eu.bauerj.paperless_app) and subscribe to this discussion (over there 🢆).

We will post here for any future beta release so you know what to look out for.

Thanks!

**Comment by mvdkleijn** [comment]
Don't use Google Play anymore, but good that you're doing a beta! :smile:

**Reply by bauerj** [reply]
Well, F-Droid users currently also receive beta releases 😅
